proto/birthday.proto

```diff
@@ -5,66 +5,72 @@ option go_package = "github.com/yarinBenisty/birthday-service/proto/birthday.pro
 package birthday;
 
 service BirthdayFunctions {
-    // CreateBirthday creates a new birthday and returns it.
-    rpc CreateBirthday(CreateBirthdayRequest) returns (BirthdayObject) {}
+  // CreateBirthday creates a new birthday and returns it.
+  rpc CreateBirthday(CreateBirthdayRequest) returns (BirthdayObject) {}
 
-    // GetBirthday returns the selected birthday.
-    rpc GetBirthday(GetBirthdayRequest) returns (BirthdayObject) {}
+  // GetBirthday returns the selected birthday.
+  rpc GetBirthday(GetBirthdayRequest) returns (BirthdayObject) {}
 
-    //UpdateBirthday updates the selected birthday (overrides the birthday info)
-    rpc UpdateBirthday(UpdateBirthdayRequest) returns (BirthdayObject) {}
+  // GetBirthday returns all birthdays.
+  rpc GetAllBirthday(GetAllBirthdayRequest) returns (GetAllBirthdayResponse) {}
 
-    //DeleteBirthday deletes the selected birthday.
-    rpc DeleteBirthday(DeleteBirthdayRequest) returns (DeleteBirthdayResponse) {}
+  // UpdateBirthday updates the selected birthday (overrides the birthday info)
+  rpc UpdateBirthday(UpdateBirthdayRequest) returns (BirthdayObject) {}
 
+  // DeleteBirthday deletes the selected birthday.
+  rpc DeleteBirthday(DeleteBirthdayRequest) returns (DeleteBirthdayResponse) {}
 }
 
 message CreateBirthdayRequest {
-    //the name of the person
-    string name = 1;
+  // the name of the person
+  string name = 1;
 
-    //the date of birth of the person
-    string date = 2;
+  // the date of birth of the person
+  string date = 2;
 
-    //the identification number of the person
-    string personalNumber = 3;
+  // the identification number of the person
+  string personalNumber = 3;
 }
 
-message DeleteBirthdayResponse{
-
-}
+message DeleteBirthdayResponse {}
 
 message GetBirthdayRequest {
-    //the identification number of the person
-    string personalNumber = 1;
+  // the identification number of the person
+  string personalNumber = 1;
 }
 
+message GetAllBirthdayRequest {}
+
 message UpdateBirthdayRequest {
-    //the identification number of the person
-    string personalNumber = 1;
+  // the identification number of the person
+  string personalNumber = 1;
 
-    //the name of the person
-    string name = 2;
+  // the name of the person
+  string name = 2;
 
-    //the date of birth of the person
-    string date = 3;
+  // the date of birth of the person
+  string date = 3;
```

Comment:
Why is date not int64?

Reply:
done.
the reason is, we agreed on from the start that it'll be easier to get and pass on both ways strings of dates.